css/desktop-layout.css

```diff
@@ -0,0 +1,78 @@
+nav {
+  height: 4.5rem;
+  padding: 1rem 4rem;
+}
+
+.header-nav img {
+  display: none;
+}
+
+.nav {
+  display: flex;
+  padding: 0.625rem 0.75rem;
+  align-items: center;
+  gap: 0.625rem;
+}
+
+.intro-hero {
+  margin: 0 auto;
+  max-width: 40rem;
+}
+
+.work {
+  padding: 8rem 5rem;
+}
+
+.card {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-areas: 'img content';
+  gap: 1.3rem;
+}
+
+.card-img {
+  grid-area: img;
+  height: 100%;
+}
+
+.card-info {
+  grid-area: content;
+  padding: 1.5rem;
+}
+
+.card:nth-child(even) {
+  grid-template-areas: 'content img';
+}
```

Comment:
The length of the cards seems to be longer than the actual design at certain widths hence it seems to take a square shape instead of the rectangular shape as shown below. Kindly consider making the shape similar to the original design as shown below. 

### Your design

![Screenshot from 2023-07-14 18-36-01](https://github.com/IsaMarvin/Portfolio_Microverse/assets/65233860/cab146c7-38dd-4841-b602-2c0552def2cc)


### Actual design 

![Screenshot from 2023-07-14 18-35-27](https://github.com/IsaMarvin/Portfolio_Microverse/assets/65233860/f966f063-453f-4d9b-a1f4-f65118a8e524)